### PR TITLE
docs: Update POSTMAN collection and add Export README instructions

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:
FHIR-428

Description of changes:
* Update POSTMAN collection to use Oauth2
* Update POSTMAN collection with requests for Bundles and Export
* Update README with instructions about Bulk Data Export

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: postman/Fhir.postman_collection.json [hunk 1]
```diff
@@ -124,11 +172,46 @@
...
+									"type": "string"
+								},
+								{
+									"key": "scope",
+									"value": "profile openid",
+									"type": "string"
+								},
+								{
+									"key": "clientId",
+									"value": "{{CLIENT_ID}}",
```
Comment (carvantes) [inline postman/Fhir.postman_collection.json:194]: We should add to the docs how to set the value for the new variable  `CLIENT_ID`

I couldn't find it on our docs. Maybe I missed it. Before we were using the `COGNITO_TOKEN` variable

File: postman/Fhir.postman_collection.json [hunk 2]
```diff
@@ -124,11 +172,46 @@
...
+									"type": "string"
+								},
+								{
+									"key": "grant_type",
+									"value": "implicit",
+									"type": "string"
+								},
+								{
+									"key": "authUrl",
+									"value": "{{AUTH_URL}}",
```
Comment (carvantes) [inline postman/Fhir.postman_collection.json:204]: Same as above. Add docs for setting the `AUTH_URL` variable

**Review by carvantes — CHANGES_REQUESTED** [review]

File: postman/Fhir.postman_collection.json (hunk 1 above)
Comment (nguyen102) [reply postman/Fhir.postman_collection.json:194]: updated

File: postman/Fhir.postman_collection.json (hunk 2 above)
Comment (nguyen102) [reply postman/Fhir.postman_collection.json:204]: updated

**Review by carvantes — APPROVED** [review]

**Pull request merged**